core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -667,6 +701,26 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     } yield savedGroup
   }
 
+  private def updateGroupMembershipInternalPolicy(groupRef: RawlsGroupRef)(update: RawlsGroup => RawlsGroup): Future[RawlsGroup] = {
```

Comment:
(this code duplication will go away, just an interim approach to see if it works)